components.json

```diff
@@ -11,7 +11,7 @@
     "prefix": ""
   },
   "aliases": {
-    "components": "@/components",
-    "utils": "@/lib/utils"
+    "components": "~/components",
+    "utils": "~/lib/utils"
```

Comment:
the root cause of why it generated `@` on the root directory instead of on the `app/components` directory